actionmailbox/lib/action_mailbox/engine.rb

```diff
@@ -26,6 +27,7 @@ class Engine < Rails::Engine
         ActionMailbox.incinerate = app.config.action_mailbox.incinerate.nil? ? true : app.config.action_mailbox.incinerate
         ActionMailbox.incinerate_after = app.config.action_mailbox.incinerate_after || 30.days
         ActionMailbox.queues = app.config.action_mailbox.queues || {}
+        ActionMailbox.amazon = app.config.action_mailbox.amazon || {}
         ActionMailbox.ingress = app.config.action_mailbox.ingress
```

Comment:
Side note: feels surprising that we can't specify an ingress _and_ pass its config. The other ingresses rely on accessing Rails credentials directly or ENV vars, when arguably the app should be able to make that decision.

Reply:
@jeremy I don't fully understand what you mean here, but I have re-worked things a bit so let me know if you still think something should be done here.